linux-static/linux-static-arm64.Dockerfile

```diff
@@ -10,14 +10,19 @@ COPY ./aarch64-musl-config.mak /tmp/config.mak
 # Build cross-compiling toolchain using https://github.com/richfelker/musl-cross-make,
 # https://github.com/rust-cross/rust-musl-cross.
 # See also `../cross/linux-musl-arm.Dockerfile`.
-RUN cd /tmp && \
+RUN set -x && \
```

Comment:
Leftovers from debugging, but I guess it's fine to leave it, might come in handy in the future.

Reply:
Exactly, I will revert that file before merging.

Reply:
It won't be that useful, because seeing the commands executed was just misleading anyway. What would be more helpful is more visibly failing on a curl download error.

Reply:
true

Reply:
Patching the `Makefile` coming from the `musl-cross-make` package (cloned during the build) would be complex and fail later.

Failing early can be implemented simpler by just trying to download that file at the start of our build. That's what I'm adding, since patching the `Makefile` which comes from the `musl-cross-make` package at build time would be more complex and failing later. Small downside is that the file will be downloaded more than once.